app/components/plan/Workout.js

```diff
@@ -29,13 +30,18 @@ export default ({ treatment, num }) => {
         {`: ${treatment.notes ? treatment.notes : 'No notes specified'}`}
       </p>
       <div className="col-xs-2 workout-button">
-        <StepsActionButton disabled={ completedWorkout ? true : false }>
-          {
-            completedWorkout ?
-              <img src={require(`../../../src/images/emojis/${completedWorkout.pain}pain.svg`)} /> :
+      {
+        completedWorkout ?
+          <StepsActionButton disabled={true}>
+            {/* TODO: Currently defaulting to pain level 3 when no input -> need real default */}
```

Comment:
I like the notion of 0 indicating no answer, and we find an emoticon for that case